### ISSUE TITLE
Flash light not working after canceling the first photo taken. 

### ISSUE THREAD

**Issue by cclaflin89** [AlexLittlejohn/ALCameraViewController#306]
Current version: 3.0.3

When taking a photo from the camera, the flashlight is working.
But if I cancel the first photo and retake, the flashlight is not working.

**Comment by huangxianhui001** [comment]
Yes, and there is the same problem, the camera cannot shoot the picture
